helix-term/.gitignore

```diff
@@ -1 +1,4 @@
 /target
+
+# This folder is used by `test_explorer` to create temporary folders needed for testing
+test_explorer
```

Comment:
I don't think it is a good idea to put this `test_explorer` here, maybe one can just reuse the existing .gitignore and test it on `target/` instead?

Reply:
Using existing folders that are gitignored and not intended to store temporary files generated during testing will likely confuse future developers, in my opinion.

Why do you think it's not a good idea to put them in a dedicated gitginored folder?

Reply:
Because it's non-standard, and I don't think people go to check `.gitignore` just to know what other developers are using to test.

Usually, I either test on `target/`, or I test on something that I have specifically excluded locally as specified in `.git/info/exclude`.

Reply:
@archseer What do you think?